inst/startup/worker.R

```diff
@@ -93,7 +93,10 @@ for (package in azbatchenv$packages) {
 }
 
 for (package in azbatchenv$github) {
-  library(package, character.only = TRUE)
+  packageDirectory <- strsplit(package, "/")[[1]]
```

Comment:
Will this handle folder1/folder2/folde3/... pattern?

Reply:
Yes, it will grab the last name in the string